app/views/archived/petitions/_petition.json.jbuilder

```diff
@@ -5,6 +5,13 @@ json.links do
   json.self archived_petition_url(petition, format: :json)
 end if defined?(is_collection)
 
+json.parliament do
+  json.period petition.parliament.period
```

Comment:
We should include the `dissolution_at` timestamp which is in the parliaments/show template